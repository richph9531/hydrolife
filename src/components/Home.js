import React from 'react';

import classes from '../styles/Home.module.css';

const Home = () => (
  <div className={classes.Home}>
    <div className={classes.HomeContent}>
      <img 
        src={`${process.env.PUBLIC_URL}/media/logo.png`}
        alt='Hydrolife Logo'/>
      <p>We have over 20 years of experience in the rental of water coolers and the delivery of refill bottled water in the greater Cape Town area. Hydrolife has the solution to your home, office or factory/plant water needs.</p>
      <p>Our expertise in off-grid solutions and bespoke water filtration systems is what sets Hydrolife apart from other excellent quality water solutions.</p>
    </div>
    <div className={classes.HomeCoolerImageContainer} >
      <img className={classes.HomeCoolerImage} 
        src={`${process.env.PUBLIC_URL}/media/cooler-tall.png`}
         alt='Rent one of these'/>
    </div>
  </div>


);

export default Home;
