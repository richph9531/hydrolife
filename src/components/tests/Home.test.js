import React from 'react';

import { render } from '@testing-library/react';
import Home from '../Home';

test('Home renders correctly', () => {
  const view = render(<Home />);
  expect(view.baseElement).toMatchSnapshot();
});
