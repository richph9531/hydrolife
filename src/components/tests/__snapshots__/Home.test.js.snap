// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<body>
  <div>
    <div
      class="Home"
    >
      <div
        class="HomeContent"
      >
        <img
          alt="Hydrolife Logo"
          src="/media/logo.png"
        />
        <p>
          We have over 20 years of experience in the rental of water coolers and the delivery of refill bottled water in the greater Cape Town area. Hydrolife has the solution to your home, office or factory/plant water needs.
        </p>
        <p>
          Our expertise in off-grid solutions and bespoke water filtration systems is what sets Hydrolife apart from other excellent quality water solutions.
        </p>
      </div>
      <img
        alt="Rent one of these"
        class="HomeCoolerImage"
        src="/media/cooler-transparent.png"
      />
    </div>
  </div>
</body>
`;
